ontology_api/tests/test_create_model.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+import unittest
+import unittest.mock as mock
+
+from functions.functions import create_model
+from functions.functions import models
+
+
+class CreateModelTestCase(unittest.TestCase):
+
+
+    def test_create_model_without_error(self):
+        my_path = "tests\\test_owl\\owlAC_testParticipant.owl"
```

Comment:
It's not portable, doesn't work on Linux.

Reply:
Do you have a suggestion how we could make it compatible with Linux?